src/grammar.coffee

```diff
@@ -429,12 +436,7 @@ grammar =
     o 'Value OptFuncExist String',              -> new TaggedTemplateCall $1, $3, $2
     o 'Value OptFuncExist Arguments',           -> new Call $1, $3, $2
     o 'Invocation OptFuncExist Arguments',      -> new Call $1, $3, $2
-    o 'Super'
-  ]
-
-  Super: [
-    o 'SUPER',                                  -> new SuperCall
-    o 'SUPER Arguments',                        -> new SuperCall $2
+    o 'SUPER OptFuncExist Arguments',           -> new SuperCall LOC(1)(new Super), $3, $2
```

Comment:
Whoa there — I don't know about all of the ES6 grammar rules and wrinkles ... but wouldn't it make more sense to bring `SUPER` in as a `Value`, instead of special-casing all of the different ways you might access it?